src/Components/Facility/Consultations/components/LinePlot.tsx

```diff
@@ -225,11 +228,18 @@ export const LinePlot = (props: any) => {
   }
 
   return (
-    <ReactEchartsCore
-      echarts={echarts}
-      option={generalOptions}
-      className={props.classes}
-      lazyUpdate={props.type === "WAVEFORM"}
-    />
+    <>
+      {yData.map((value: any) => (
+        <span className="sr-only px-1">
```

Comment:
LGTM, Could you give the span an `id` too so that Cypress can uniquely identify the element to read?

Reply:
ya sure